burrowers/garble
# ssaReflect

- scripts: remove TODO about building third party programs
- CI: test against a newer Go tip
- Add pagran to FUNDING.yml
- use fewer build flags when building std or cmd
- use git in english when matching its output
- use the host's GOEXE when building the linker
- avoid panic when a package's imports cannot be loaded
- ensure the alignment of sync/atomic types works
- update gotip and adapt to upstream changes
- avoid rebuilding garble in the main benchmark
- print chosen seed when building with -seed=random
- support `garble run`
- update gotip in CI and fix -tiny on the latest tip
- use the "simple" obfuscator for large literals
- update to Go tip from April 28th
- set testscript's RequireExplicitExec and RequireUniqueNames
- go.mod: bump deps
- scripts: no need for this directory to be a Go package
- CI: skip regular `go test` on linux
- add a test to reproduce our caching bugs
- fix encoding/asn1 marshaling of pkix types
- correctly detect indirect uses of reflection
- detect which objects are global in a simpler way
- avoid breaking github.com/davecgh/go-spew
- rework reflection detection with ssa
